docs/dev/libvirt/README.md

```diff
@@ -247,6 +247,31 @@ This step allows installer and users to resolve cluster-internal hostnames from
 
 3. Reload NetworkManager to pick up the `dns` configuration change: `sudo systemctl reload NetworkManager`
 
+### Setting the resolver on Fedora 34+ with systemd-resolved
+
+Fedora 34 switched to utilizing systemd-resolved and changes how DNS resolution works on a per-interface basis. 
+
+If you are running Fedora 34, you must indicate that you want your resolver to point to the local dnsmasq resolver on the `tt0` interface.
+
+The below example uses the default `tt0` interface, change the interface name if you are utilizing a custom interface/bridge instead.
+
+```sh
+resolvectl dns tt0 127.0.0.1
+```
+
+You may see which interfaces point to which DNS resolvers by typing `resolvectl`. `tt0` should be now pointing to 127.0.0.1, where the dnsmasq process is listening.
+
```

Comment:
Does this configuration survives a destroy/create sequence (when the libvirt network is destroyed/recreated)?

Reply:
It does not, this needs to be re-created every time.  I will most likely change this section favoring the method outlined in your comment above https://github.com/openshift/installer/pull/4929#discussion_r633562305

Reply:
Did you have a chance to try https://github.com/openshift/installer/pull/4929#discussion_r633562305 ?
If not/if it did not work, we can go with this PR, but mentioning that this change is not persistent would be useful.

Reply:
I would like to address this in a separate PR to simplify how networking is done for libvirt development setups.  I was not able to get a working setup without dnsmasq running on the host. I think there could be some discussion on what the best architecture would be to make sure it is more automated in terms of deployment.